scripts/pysa_fuzzer/run.py

```diff
@@ -0,0 +1,150 @@
+import subprocess
+import os
+import json
+import argparse
+from code_generator import CodeGenerator
+from pathlib import Path
+import shutil
+import logging
+
+logging.basicConfig(level=logging.INFO)
+
+def run_command(command, output_file=None):
+    try:
+        result = subprocess.run(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True, check=True)
+        if output_file:
+            with open(output_file, 'w') as file:
+                file.write(result.stdout)
+    except subprocess.CalledProcessError as e:
+        logging.error(f"Command '{command}' failed with error: {e.stderr}")
+
+def generate_python_files(num_files, num_statements):
+    generator = CodeGenerator()
+    output_dir = Path('generated_files')
+
+    output_dir.mkdir(exist_ok=True)
+
+    filenames = []
+    for i in range(1, num_files + 1):
+        generator.reset()
+        generated_code = generator.generate_statements(num_statements)
+        filename = output_dir / f'test_{i}.py'
+        with open(filename, 'w') as file:
+            file.write(generated_code)
+        filenames.append(str(filename))
+        logging.info(f"Generated {filename}")
+
+    # Save the filenames to a temporary file
+    with open('filenames.tmp', 'w') as tmp_file:
+        json.dump(filenames, tmp_file)
+
+def configure_and_analyze():
+    with open('.pyre_configuration', 'w') as config_file:

```

Comment:
can we actually create all the generated files in a directory `generated/`? 
This will also make the implementation of `clean_up` easy, we can use `shutil.rmtree` to delete the directory.